low-level/files/opt/seagate/sspl/setup/sspl_post_install.py

```diff
@@ -103,7 +104,17 @@ def validate(self):
             # cluster_id required for decrypting the secret is only available from
             # the prepare stage. However accessibility validation will be done in
             # prepare stage. So at this time, validating ip reachability is fine.
-            NetworkV().validate("connectivity", [bmc_ip, primary_ip, secondary_ip])
+            for ip in [bmc_ip, primary_ip, secondary_ip]:
```

Comment:
I thought we decided to extend validators itself, taking these retry count, sleep time as inputs. So others can also take advantage !?
For quick fix in SSPL, this is fine obviously

Reply:
`ping -c 1 -W 1 <ip>` is the cmd used in ip connectivity check.
Even if we increase ping `-c 1` to `-c 50`, exception will be thrown for one of the packet loss.
So I feel retry from SSPL side is better here.

Reply:
Hi @mariyappanp, there is one option `-w (small w not capital)` which is a deadline, so for example we can have a deadline of 4sec `-w 4` and count of 2 `-c 2`. for this config ping will wait for 4 sec to get 2 packets successfully sent, if 2 pkts are sent before the deadline then ping will succeed even if 1,2 or 3, etc pkts are dropped.

Is it possible to use this❔

Reply:
![image](https://user-images.githubusercontent.com/45289079/129717857-1aaa3c0d-1bc9-42a0-851b-4498e1dd99ee.png)

Reply:
Possible in cortx-utils. But still SSPL will have minor change to pass it.
So two PR will be needed in that case.

Reply:
@sumedhak27 , current approach with SimpleProcess in cortx-utils is same.
If return code is not 0, its reachable. If non-zero, then unreachable.

Reply:
yes agree @mariyappanp , we will go ahead with changes in SSPL itself right now